firefox_ui_tests/firefox_ui_tests/functional/security/test_safe_browsing_primary_download.py

```diff
@@ -0,0 +1,88 @@
+import os
+import sys
+
+from marionette_driver import By, Wait
+from marionette.marionette_test import skip_if_e10s
```

Comment:
This skip is indeed not necessary if the test works in e10s mode. If Travis passes without it, we can be sure about it.